### ISSUE TITLE
Feature request: Providing easier access to Prisma Client

### ISSUE THREAD

**Issue by maoosi** [maoosi/prisma-appsync#7]
## Problem

Right now, accessing the Prisma client instance is only possible via the hooks functions. This makes it unhandy to use utility methods such as Prisma middleware or events.

## Suggested solution

Expose Prisma client instance via `[instanceof PrismaAppSync].prisma`, such as:

```typescript
// initialise client
const app = new PrismaAppSync({
    connectionUrl: process.env.CONNECTION_URL
})

// access Prisma client
app.prisma.$use(async (params, next) => {
  console.log('This is middleware!')
  return next(params)
})
```

## Additional context

- Most of the work will probably happen within `src/client/index.ts` (main class) and `src/client/_resolver.ts` (where Prisma Client is currently initiated).
- **Open discussion:** Should we remove [`app.$disconnect`](https://prisma-appsync.vercel.app/reference/client-api.html#disconnect) entirely and use `app.prisma.$disconnect` instead? Or do we think it is more accessible this way, and should only push `app.prisma.xxx` usage for more "advanced scenarios"? I'm thinking option 2, but keen to get extra opinions on this.
- **Open discussion:** Similarly, should we remove the `prisma` parameter from the before and after hooks functions? I'm thinking yes since using Prisma client directly would probably fit into "advanced scenario". But again, keen to get extra opinions on this.

**Comment by DregondRahl** [comment]
I think putting it under app.prisma would be a good option, we can easily extend any middleware and also by keeping the syntax consistent it would be easier for a developer to know he can access prisma from app anywhere he needs to.

**Comment by maoosi** [comment]
Resolved with [1.0.0-beta.52](https://github.com/maoosi/prisma-appsync/releases/tag/1.0.0-beta.52)

⚠️ This introduces some breaking changes, so make sure to read the Changelog and update your project before upgrading.
